### PR TITLE
Bump Version

### PR DESCRIPTION
**Pull request by alexpattyn**
Based on the changes made in https://github.com/flathub/fr.free.Homebank/pull/11

### CODE REVIEW

File: fr.free.Homebank.json [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
     "sdk": "org.gnome.Sdk",
     "command": "homebank",
     "rename-icon": "homebank",
     "copy-icon": true,
     "rename-desktop-file": "homebank.desktop",
     "rename-appdata-file": "homebank.appdata.xml",
     "finish-args": [
-            "--share=ipc", "--socket=x11",
+            "--share=ipc", 
+            "--socket=x11",
```
Comment (bilelmoussaoui) [inline fr.free.Homebank.json:13]: This should be migrated to fallback-x11 if it really works under wayland. You can try that by running the app with `flatpak run --nosocket=x11 fr.free.Homebank` under a wayland session.

Comment (alexpattyn) [reply fr.free.Homebank.json:13]: Testing the current version I have installed from flathub running `flatpak run --nosocket=x11 fr.free.Homebank` works. I ran it on fedora 24 beta with wayland. So I should just change `--socket=x11` to `--socket=fallback-x11`?

Comment (alexpattyn) [reply fr.free.Homebank.json:13]: ^ That would be a yes according to https://docs.flatpak.org/en/latest/sandbox-permissions.html?highlight=fallback-x11#standard-permissions

Comment (bilelmoussaoui) [reply fr.free.Homebank.json:13]: Yes please, socket=x11 gives the app access to the x11 socket no matter whether it runs under wayland or not. While the fallback-x11 only gives it to the app when it fails to reach a wayland socket. It's a stricter permission.

**Comment by alexpattyn** [comment]
Just tested the latest build. Looks good on my end.

**Comment by bilelmoussaoui** [comment]
Thanks!

**Comment by alexpattyn** [comment]
No problem! Thanks for your help!

**Pull request merged**